spec.html

```diff
@@ -37,6 +37,143 @@ <h1>Introduction</h1>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error">
+        <h1>AggregateError ( _errors_, _message_ )</h1>
+        <p>When the *AggregateError* function is called with arguments _errors_ and _message_, the following steps are taken:</p>
+        <emu-alg>
+          1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+          1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+          1. Let _errorsArray_ be ! CreateArrayFromList(? IterableToList(_errors_)).
+          1. Perform ! CreateMethodProperty(_O_, `"errors"`, _errorsArray_).
+          1. If _message_ is not _undefined_, then
+            1. Let msg be ? ToString(_message_).
+            1. Perform ! CreateMethodProperty(_O_, `"message"`, _msg_).
+          1. Return _O_.
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregate-error-constructors">
+      <h1>Properties of the _AggregateError_ Constructor</h1>
+      <p>The _AggregateError_ constructor:</p>
+      <ul>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error%.</li>
+        <li>has a `name` property whose value is the String value `"<var>AggregateError</var>"`.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype">
+        <h1>AggregateError.prototype</h1>
+        <p>The initial value of `AggregateError.prototype`  is the intrinsic object %AggregateErrorPrototype%.</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregate-error-prototype-objects">
+      <h1>Properties of the _AggregateError_ Prototype Object</h1>
+      <p>The _AggregateError_ prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype.constructor">
+        <h1>_AggregateError_.prototype.constructor</h1>
+        <p>The initial value of `AggregateError.prototype.constructor` is the intrinsic object %_AggregateError_% (<emu-xref href="#sec-aggregate-error-constructors"></emu-xref>).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.name">
+        <h1>_AggregateError_.prototype.name</h1>
+        <p>The initial value of `AggregateError.prototype.name` is "AggregateError".</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.errors">
+        <h1>_AggregateError_.prototype.errors</h1>
```

Comment:
repeating: there must not be a mutable object data property on the prototype.

`AggregateError.prototype.errors` must either be absent, a primitive, an accessor that returns a new array every time (either empty, or with the errors if #38 moves forward), or a data property using a frozen array.

Reply:
Should it be added to `Properties of the _AggregateError_ Constructor` part?